creator/image.go

```diff
@@ -158,6 +158,13 @@ func (img *Image) GetMargins() (float64, float64, float64, float64) {
 	return img.margins.left, img.margins.right, img.margins.top, img.margins.bottom
 }
 
+// ToBinaryImage converts img data into Binary image (Bi-level image).
+// If provided image is RGB or GrayScale the function converts it into binary image
+// using histogram auto threshold method.
+func (img *Image) ToBinaryImage() error {
```

Comment:
Maybe we should call this `ToBinary` or `ConvertToBinary` instead of `ToBinaryImage`.

Reply:
I think `ConvertToBinary`  describes the function very well. Done.